hydra-tui/src/Hydra/TUI.hs

```diff
@@ -232,6 +234,16 @@ handleEvent client@Client{sendInput} cardanoClient s = \case
   e ->
     continue $ s & warn ("unhandled event: " <> show e)
 
+doSendInputAndTransitionIfNotPending :: Client tx IO -> State -> ClientInput tx -> EventM n (Next State)
```

Comment:
suggestion: Rename to something less a mouthful, perhaps taking as argument the pending state?
```
s ^? pendingL >>= transition

transition :: Client tx IO -> State -> ClientInput tx ->  Wait -> EventM n (Next State)
....
```
Not sure this is really helpful 🤔

Reply:
renamed to `sendInputAndTransition`